syntax = "proto3";

service EmotionService {
    rpc GetTextEmotion(Text) returns (EmotionResponse) {};
}

message Text {
  string text = 1;
}

message EmotionResponse {
    map<string, float> scores = 1;
}